OsmAnd/src/net/osmand/plus/settings/backend/backup/exporttype/ExportType.java

```diff
@@ -0,0 +1,184 @@
+package net.osmand.plus.settings.backend.backup.exporttype;
+
+import static net.osmand.plus.settings.backend.backup.exporttype.AbstractMapExportType.OLD_OFFLINE_MAPS_EXPORT_TYPE_KEY;
+import static net.osmand.util.Algorithms.addAllIfNotContains;
+import static net.osmand.util.Algorithms.addIfNotContains;
+import static net.osmand.util.Algorithms.filterElementsWithCondition;
+import static net.osmand.util.Algorithms.searchElementWithCondition;
+
+import androidx.annotation.DrawableRes;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.StringRes;
+
+import net.osmand.plus.OsmandApplication;
+import net.osmand.plus.backup.RemoteFile;
+import net.osmand.plus.download.local.LocalItemType;
+import net.osmand.plus.plugins.OsmandPlugin;
+import net.osmand.plus.plugins.PluginsHelper;
+import net.osmand.plus.settings.backend.ExportCategory;
+import net.osmand.plus.settings.backend.backup.SettingsItemType;
+import net.osmand.plus.settings.backend.backup.items.FileSettingsItem;
+import net.osmand.plus.settings.backend.backup.items.FileSettingsItem.FileSubtype;
+import net.osmand.plus.settings.backend.backup.items.SettingsItem;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public enum ExportType {
+
+	PROFILE(new ProfileExportType()),
```

Comment:
Do we need enum after these changes?  What if we use static instances similar to [ApplicationMode ](https://github.com/osmandapp/Osmand/blob/9ddfd7abc343aa39803a6c5604c3bdab60a2c14d/OsmAnd/src/net/osmand/plus/settings/backend/ApplicationMode.java#L41-L111)